pkg/build/client/internalversion/buildlogs.go

```diff
@@ -0,0 +1,32 @@
+package internalversion
+
+import (
+	restclient "k8s.io/client-go/rest"
+	kapi "k8s.io/kubernetes/pkg/api"
+
+	buildapi "github.com/openshift/origin/pkg/build/apis/build"
+	buildinternalversion "github.com/openshift/origin/pkg/build/generated/internalclientset/typed/build/internalversion"
+)
+
+type BuildLogInterface interface {
+	Logs(name string, options buildapi.BuildLogOptions) *restclient.Request
+}
+
+func NewBuildLogClient(c buildinternalversion.BuildInterface, ns string) BuildLogInterface {
+	return &buildLogs{client: c, ns: ns}
+}
+
+type buildLogs struct {
+	client buildinternalversion.BuildInterface
```

Comment:
Looking at the package dependencies, I think this is going to have to be a `rest.Interface`

Reply:
> Looking at the package dependencies, I think this is going to have to be a rest.Interface

still oustanding